Gemfile.lock

```diff
@@ -347,33 +342,33 @@ GEM
       netrc (~> 0.8)
     rspec-core (3.8.2)
       rspec-support (~> 3.8.0)
-    rspec-expectations (3.8.4)
+    rspec-expectations (3.8.6)
```

Comment:
### [rspec-expectations](https://github.com/rspec/rspec-expectations)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.8.4 → 3.8.6

[_[change-log](https://github.com/rspec/rspec-expectations/blob/v3.8.6/Changelog.md), [source-code](https://github.com/rspec/rspec-expectations)_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-expectations/compare/v3.8.4...v3.8.6).

These are the individual commits:
- (a86409f) [Fix ffi on Ruby 2](https://github.com/rspec/rspec-expectations/commit/a86409f855c85b8bb692c4dc1b04717d6aad8744)
- (a6c148c) [Update Aruba to version 0.14.10 (#1118)](https://github.com/rspec/rspec-expectations/commit/a6c148cff550a16209dac724b75a294f19fb544e)
- (4ff7f10) [Add Expectations namespace to ExpectationTarget tags. (#1121…](https://github.com/rspec/rspec-expectations/commit/4ff7f10275c02054c35422a0807e90298dc216db)
- (8076960) [Updated travis build scripts (from rspec-dev) (#1122)](https://github.com/rspec/rspec-expectations/commit/80769609c55aab93afe89a2a5c7a104dff016484)
- (3ab59e2) [Updated travis build scripts (from rspec-dev) (#1123)](https://github.com/rspec/rspec-expectations/commit/3ab59e20b34b763dcb09029a33114cba60fcbea1)
- (f5bf9d8) [Merge pull request #1125 from pirj/prevent-block-matchers-fr…](https://github.com/rspec/rspec-expectations/commit/f5bf9d8264ffd786a66fe25c3bb1125c93b59d89)
- (946fe54) [Pin minitest on older Rubies (#1133)](https://github.com/rspec/rspec-expectations/commit/946fe5485bc4546d30c285f60f33627799ac167f)
- (a85bbd6) [Version 3.8.5](https://github.com/rspec/rspec-expectations/commit/a85bbd6854e4ced5eef9e01f6279c4b90be8d307)
- (9423442) [Revert "Merge pull request #1125 from pirj/prevent-block-mat…](https://github.com/rspec/rspec-expectations/commit/94234426ff70244f3f388ecee944e8954cff2e93)
- (f7c4a6f) [Version 3.8.6](https://github.com/rspec/rspec-expectations/commit/f7c4a6f38820d2cae81687bc6f5ccc9626bc3418)


</details>